coalib/coala_main.py

```diff
@@ -180,7 +180,13 @@ def run_coala(console_printer=None,
             if not autoapply:
                 section['default_actions'] = ''
             elif force_show_patch:
-                section['default_actions'] = '*: ShowPatchAction'
+                # Append ShowPatchAction to default actions
+                try:
+                    default_actions = str(section['default_actions'])
+                except IndexError:
```

Comment:
the `except` can be accomplished in one single line i.e. `section['default_actions'] = '*: ShowPatchAction'`
because, in this `except block` you are appending an empty string to `default_actions`

Reply:
Didn't notice. Thanks.